### PR TITLE
317: refactor - rename judge notes and procedural notes for clarity

### PR DESCRIPTION
**Pull request by rachaelparris**
case notes -> judge's notes
procedural notes -> case notes

### CODE REVIEW

**Review by codyseibert — APPROVED** [review]

File: shared/src/business/useCases/caseNote/saveCaseNoteInteractor.js [hunk 1]
```diff
@@ -6,18 +6,18 @@ const { Case } = require('../../entities/cases/Case');
 const { UnauthorizedError } = require('../../../errors/errors');
 
 /**
- * saveProceduralNoteInteractor
+ * saveCaseNoteInteractor
  *
  * @param {object} providers the providers object
  * @param {object} providers.applicationContext the application context
  * @param {string} providers.caseId the id of the case to update procedural note
  * @param {string} providers.notes the note to update
```
Comment (kkoskelin) [inline shared/src/business/useCases/caseNote/saveCaseNoteInteractor.js]: deconstructed arg name mismatch

File: web-client/src/presenter/actions/TrialSession/deleteJudgesCaseNoteAction.js [hunk 2]
```diff
@@ -1,22 +1,25 @@
 /**
- * delete a case note
+ * delete a judge's case note
  *
  * @param {object} providers the providers object
  * @param {object} providers.applicationContext the application context
  * @param {object} providers.props the cerebral props object
  * @param {object} providers.get the cerebral get function
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/TrialSession/deleteJudgesCaseNoteAction.js]: no `get` here.

File: web-client/src/presenter/actions/TrialSession/getJudgesCaseNoteForCaseAction.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { state } from 'cerebral';
+
+/**
+ * Fetches the judge's case note for a case
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext needed for getting the getTrialSessionDetails use case
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/TrialSession/getJudgesCaseNoteForCaseAction.js]: jsdoc update required

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**